tests/plotting/test_plotting.py

```diff
@@ -1651,7 +1651,8 @@ def test_opacity_by_array_uncertainty(uniform):
 
 
 def test_opacity_by_array_user_transform(uniform):
-    uniform['Spatial Point Data'] /= uniform['Spatial Point Data'].max()
+    scaled = uniform['Spatial Point Data'] / uniform['Spatial Point Data'].max()
+    uniform['Spatial Point Data'] = scaled
```

Comment:
Can this be reverted now that we've fixed return type and reference issues?

Reply:
Correct. This has been cleaned up.